src/doc/rustdoc/src/how-to-read-rustdoc.md

```diff
@@ -0,0 +1,101 @@
+# How to read rustdoc output
+
+Rustdoc's HTML output includes a friendly and useful navigation interface,
+to make it easier for users to navigate and understand your code.
+This chapter covers the major features of that interface,
+and is a great starting point for documentation authors and users alike.
+
+## Structure
+
+The `rustdoc` output is divided into three sections.
+Along the left side of each page is a quick navigation bar,
+which shows contextual information about the current entry.
+The rest of the is taken up by the search interface at the top
+and the documentation for the current item below that.
+
+## The Item Documentation
+
+The majority of the screen is taken up with the main event:
+the documentation text for the item currently being viewed.
+The main title is the type and name of the item,
+such as "Struct `std::time::Duration`",
+along with a button to collapse or expand the top-level documentation for that
+item (`[+]` or `[-]`),
```

Comment:
@jsha If we merge this PR with this doc, we'll need to not forget to update it in case we change the collapse/expand buttons.

Reply:
Yep, makes sense! We would also need to update https://doc.rust-lang.org/std/#how-to-read-this-documentation.